hardware/EvohomeBase.h

```diff
@@ -395,8 +401,17 @@ class CEvohomeMsg
 		pktwrt,
 	};
 
-	CEvohomeMsg() :flags(0), type(pktunk), timestamp(0), command(0), payloadsize(0), readofs(0), enccount(0) {}
-	CEvohomeMsg(const char * rawmsg) :flags(0), type(pktunk), timestamp(0), command(0), payloadsize(0), readofs(0), enccount(0) { DecodePacket(rawmsg); }
+	CEvohomeMsg() = default;
+	CEvohomeMsg(const char *rawmsg)
+		: flags(0)
+		, timestamp(0)
+		, command(0)
+		, payloadsize(0)
+		, readofs(0)
+		, enccount(0)
+	{
```

Comment:
Will all of these not be initialized in the header with the PR ?

Reply:
Good catch. Will fix.